### PR TITLE
docs: fill out readme

### PR DESCRIPTION
**Pull request by francinelucca**
Related #https://github.com/ibm-telemetry/telemetry-js/issues/144
TODO:

- [x] Reference Telemetry-js project
- [x] Add data on how to obtain projectID
- [x] Add data on which endpoint to use - we will provide this internally when onboarding teams

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,152 @@
...
+"[Learn YAML in Y minutes](https://learnxinyminutes.com/docs/yaml)." Below is the latest stable
+version of the schema.
+
+## Stable version
+
+The current supported stable version of the config schema is _v1_. You can add the YAML language
+server to your YAML files by adding the following line to the top of your files:
+
+```yml
+# yaml-language-server: $schema=https://unpkg.com/@ibm/telemetry-config-schema@0.3.0/dist/config.schema.json
```
Comment (mattrosno) [inline README.md:17]: Should this be

https://unpkg.com/@ibm/telemetry-config-schema@v1/dist/config.schema.json

or

https://unpkg.com/@ibm/telemetry-config-schema@v1/dist/config.schema.json

? You can use `vX` syntax with unpkg if you want it to use the latest of that major version.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,152 @@
...
+        - 'title1'
+        - 'title2'
+```
+
+### Collect keys
+
+| Key   | Description                                                                                                       | Required | Type   |
+| ----- | ----------------------------------------------------------------------------------------------------------------- | -------- | ------ |
+| `npm` | Configuration for collecting telemetry data from an npm environment. See [npm schema](#npm-schema) for more info. | Optional | Object |
+| `jsx` | Configuration for collecting telemetry data from JSX files. See [jsx schema](#jsx-schema) for more info.          | Optional | Object |
```
Comment (mattrosno) [inline README.md:83]:
```suggestion
| `jsx` | Configuration for collecting telemetry data from JSX files. See [JSX schema](#jsx-schema) for more info.          | Optional | Object |
```

I think it's always `JSX` capitalization.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,152 @@
...
+The npm config object has a single required `dependencies` key that takes a `null` value. This key
+**must** be present inside the npm config if npm scope data is to be collected:
+
+```yaml path="sample-telemetry.yml"
+---
+npm:
+  dependencies: null
+```
+
+## Jsx schema
```
Comment (mattrosno) [inline README.md:106]:
```suggestion
## JSX schema
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,152 @@
...
+
+Determines configuration for jsx scope collection.
```
Comment (mattrosno) [inline README.md:108]:
```suggestion
Determines configuration for JSX scope collection.
```

File: README.md [hunk 5]
```diff
@@ -1,3 +1,152 @@
...
+
+The jsx scope captures (jsx) element-specific usage data for the instrumented package. Specifically:
```
Comment (mattrosno) [inline README.md:110]:
```suggestion
The JSX scope captures (JSX) element-specific usage data for the instrumented package. Specifically:
```

File: README.md [hunk 6]
```diff
@@ -1,3 +1,152 @@
...
+
+- All elements exported through the instrumented package that are being used in a given project that
+  installed the package.
+- Element configurations (attributes and values), as determined by the `allowedAttributeNames` and
+  `allowedAttributeStringValues` config options defined below.
+- Import paths used to access the instrumented package's exported elements.
+
+The jsx config object has a single required `elements` key. This is an object that may be left empty
```
Comment (mattrosno) [inline README.md:118]:
```suggestion
The JSX config object has a single required `elements` key. This is an object that may be left empty
```

File: README.md [hunk 7]
```diff
@@ -1,3 +1,152 @@
...
+  At least one string value is required if this key is defined.
+
+- `allowedAttributeStringValues`: This is an _optional_ array of String.
+
+  Enables telemetry data collection for specific string attribute values. These are collected for
+  all defined attributes in the `allowedAttributeNames` key.
+
+  At least one value is required if this key is defined.
+
+The `elements` key **must** be present inside the jsx config if jsx scope data is to be collected:
```
Comment (mattrosno) [inline README.md:137]:
```suggestion
The `elements` key **must** be present inside the jsx config if JSX scope data is to be collected:
```

**Review by mattrosno — CHANGES_REQUESTED** [review]
Do we need to share that the JSX scope is used with React packages? Our non-technical audience might not know that.

File: README.md (hunk 1 above)
Comment (francinelucca) [reply README.md]: Fixed, thanks!!

**Review by jdharvey-ibm — APPROVED** [review]

**Pull request merged**